src/model/model_input.jl

```diff
@@ -1,27 +1,5 @@
-# This file declares functions used for loading and manipulating data and sets within the
-# model file. It should be included at the beginning after loading other packages
-
-# !!!! Where might ensurefinite go within src directory?
-"This function replaces `NaN` or `Inf` values with `0.0`."
-ensurefinite(x::Float64) = (isnan(x) || x==Inf) ? 0.0 : x
-
-
-"""
-    nonzero_subset(df::DataFrame)
-
-# Returns
-- `x::Array{Tuple,1}` of all parameter indices corresponding with non-zero values
-- `idx::Array{Symbol,1}` of parameter indices in `df`
-"""
-function nonzero_subset(df::DataFrame)
-    idx = findindex(df)
-    val = convert_type(Array{Tuple}, dropzero(df)[:,idx])
-    return (val, idx)
-end
-
-
 """
-    _model_input(year::Int, d::Dict{Symbol,DataFrame}, set::Dict, idx::Dict)
+    model_input(year::Int, d::Dict{Symbol,DataFrame}, set::Dict, idx::Dict)
```

Comment:
**Enhancement** @jonmbecker's model_input functions are now canon! We can maybe rename this to something cute like a verb, but I got rid of the leading underscore since this is *not* an internal function.